### ISSUE TITLE
css generation fails

### ISSUE THREAD

**Issue by twodrops** [useblocks/sphinx-simplepdf#4]
Inside a docker container where the tool is not running under a sudo user, the following error is raised and the generation stops

>Generating css files from scss-templates
>Exception occurred:
  File "/usr/local/lib/python3.10/site-packages/sass.py", line 266, in compile_dirname
    with io.open(
PermissionError: [Errno 13] Permission denied: '/usr/local/lib/python3.10/site-packages/builders/../themes/sphinx_simplepdf/static/styles/main.css'

**Comment by danwos** [comment]
It looks like the sphinx generation gets called by a different user than the one used for installing the python packages.
This can be a general problem and can happen on each system.

Looks like we need to generate the ``main.css``` inside the output folder of the sphinx documentation so that we can be sure to have the needed rights.  
I'll try some stuff :)

**Comment by kreuzberger** [comment]
How is pdf customization planed / intended? so currently 99% fits but i would like / need some adaptions. What is the intended way? Below is the generated css section in the html file.
My "custom" fsp.css is included too early e.g. to overwrite settings in styles/main.css (but currently not generated/available) or is this the intended "personal"  css file there i have to put my changes for sphinx_simplepdf?

```
  <link href="_static/pygments.css" rel="stylesheet" type="text/css"/>
  <link href="_static/main.css" rel="stylesheet" type="text/css"/>
  <link href="_static/css/fsp.css" rel="stylesheet" type="text/css"/>
  <link href="_static/styles/main.css" rel="stylesheet" type="text/css"/>
  <link href="_static/assets/jsonview.bundle.css" rel="stylesheet" type="text/css"/>
  <link href="_static/sphinx-needs/libs/html/datatables.min.css" rel="stylesheet" type="text/css"/>
  <link href="_static/sphinx-needs/common.css" rel="stylesheet" type="text/css"/>
  <link href="_static/sphinx-needs/styles.css" rel="stylesheet" type="text/css"/>
  <link href="_static/sphinx-needs/layouts.css" rel="stylesheet" type="text/css"/>
  <link href="_static/sphinx-needs/modern.css" rel="stylesheet" type="text/css"/>
```

**Comment by kreuzberger** [comment]
So i would suggest new variables for footer/header configuration. If you would agree i would make a pull request with these variables / adapted _pages.scss:
* footer [True/False] default: False if True a "seperator" like for header is added
* header_left: string: default counter(page) #like now Have to figure out how to handle the strings stuff
* header_center: string: default empty
* header_right: string: default: string(heading) # like now
* footer_left, footer_center, footer_right: same as above

**Comment by kreuzberger** [comment]
ok, i have a closer look at the cover, just watched the _pages.scss for usage of the counter and title in the header. i will contact on further questions.

**Comment by danwos** [comment]
This should be solved by #18, which does not make any more file operations in the site-package folder of the python installation.

Please reopen, if there are still problems in docker.
